src/components/SavedNewsHeader/SavedNewsHeader.jsx

```diff
@@ -0,0 +1,28 @@
+function SavedNewsHeader({ savedArticles, keywords = [] }) {
+  const formatKeywords = (keywords) => {
+    const length = keywords.length;
+    if (length === 1) return keywords[0];
+    if (length === 2) return `${keywords[0]}, ${keywords[1]}`;
+    if (length === 3) return `${keywords[0]}, ${keywords[1]}, ${keywords[2]}`;
+    if (length > 2)
+      return `${keywords[0]}, ${keywords[1]}, and ${length - 2} other${
+        length - 2 > 1 ? "s" : ""
+      }`;
+    return "";
+  };
+
+  return (
+    <div className="saved__news-header">
+      <p className="saved__news--page">Saved articles</p>
+      <h2 className="saved__news--title">
+        Elise, you have {savedArticles.length} saved articles
+      </h2>
+      <p className="saved__keywords">
+        By keywords:{" "}
+        <span className="saved__keywords-text">{formatKeywords(keywords)}</span>{" "}
```

Comment:
You have a lot of elements of the `saved` block, but you don't have this block as the parent anywhere in the project. 

There are some `BEM` mistakes. 

You need to find them and correct according to `BEM`. 

More info about `BEM` naming can be found here: https://en.bem.info/methodology/quick-start/